packages/genesys-spark-tokens/reports/unused-defined-ui-tokens.json

```diff
@@ -0,0 +1,555 @@
+[
+  "--gse-ui-accordion-contentItem-backgroundColor",
```

Comment:
We can use this report to

- make sure our components are not missing tokens they should be using
- make suse UX has not left in tokens they are not using